javascript/test/basic-tests.js

```diff
@@ -95,5 +96,67 @@ describe('Basic Tests', function() {
 
             assert.strictEqual(exitCode, 0, "Wrong exit for valid (won) game")
         });
+
+        it('Win Minesweeper after first move', function() {
+            // Creates the simple.cfg file
+            const lines = ['..*', '...', '...'];
+            fs.writeFileSync("simple.cfg", lines.join('\n') + '\n')
```

Comment:
This test never remove the simple.cfg file and it might overwrite and existing file. Maybe you can use a temp folder and. Before/after to create and remove files? Before creates a temp file and store this value on a variable, the test fills up this file, the before remove the file to ensure the state is not polluted.

Reply:
I'm not sure if I understand the problem. I included the test in the describe function with title 'Runs with null exit code.  Tag: Assignment1, Assignment2, Assignment3, Assignment4, Assignment5' because it fits in the context. And here is already a beforeEach and afterEach included, which checks for an existing simple.cfg and removes it accordingly. Isn't this sufficient?

Reply:
Yes.